chloggen/internal/chlog/summary.go

```diff
@@ -75,5 +76,7 @@ func (s summary) String() (string, error) {
 		return "", fmt.Errorf("failed executing template: %w", err)
 	}
 
-	return buf.String(), nil
+	// Replace all \r\n with \n to ensure consistent output across platforms.
+	str := strings.ReplaceAll(buf.String(), "\r\n", "\n")
+	return str, nil
```

Comment:
GitHub Windows Runner pulls the repo with `core.autocrlf=true`, converting the template to `\r\n` line ending.
This results to the test failing
```
expected: "\r\n## 1.0\r\n\r\n### 🛑 Breaking changes 🛑\r\n\r\n- `foo`: broke foo (#123)\r\n- `bar`: broke bar (#345, #678)\r\n  more details\r\n\r\n### 🚩 Deprecations 🚩\r\n\r\n- `foo`: deprecate foo (#1234)\r\n- `bar`: deprecate bar (#3456, #6789)\r\n  more details\r\n\r\n### 🚀 New components 🚀\r\n\r\n- `foo`: new foo (#2)\r\n- `bar`: new bar (#4, #7)\r\n  more details\r\n\r\n### 💡 Enhancements 💡\r\n\r\n- `foo`: enhance foo (#12)\r\n- `bar`: enhance bar (#34, #67)\r\n  more details\r\n\r\n### 🧰 Bug fixe s 🧰\r\n\r\n- `foo`: bug foo (#1)\r\n- `bar`: bug bar (#3, #6)\r\n  more details\r\n"
actual  : "\r\n## 1.0\r\n\r\n### 🛑 Breaking changes 🛑\r\n\r\n- `foo`: broke foo (#123)\r\n- `bar`: broke bar (#345, #678)\n  more details\r\n\r\n### 🚩 Deprecations 🚩\r\n\r\n- `foo`: deprecate foo (#1234)\r\n- `bar`: deprecate bar (#3456, #6789)\n  more details\r\n\r\n### 🚀 New components 🚀\r\n\r\n- `foo`: new foo (#2)\r\n- `bar`: new bar (#4, #7)\n  more details\r\n\r\n### 💡 Enhancements 💡\r\n\r\n- `foo`: enhance foo (#12)\r\n- `bar`: enhance bar (#34, #67)\n  more details\r\n\r\n### 🧰 Bug fixes 🧰\r\n  \r\n- `foo`: bug foo (#1)\r\n- `bar`: bug bar (#3, #6)\n  more details\r\n"
```
Here we clean the output by replacing all `\r\n` to `\n`

Reply:
😢 seems like more tests are failing due to this

Reply:
Try adding https://github.com/open-telemetry/opentelemetry-go/blob/main/.gitattributes